config/config.py

```diff
@@ -11,3 +11,43 @@
     "https://api.regional-table.region-services.aws.a2z.com/index.json"
 )
 aws_services_json_file = "providers/aws/aws_regions_services.json"
+
+
+csv_header = [
+    "PROVIDER",
+    "PROFILE",
+    "ACCOUNT_ID",
+    "REGION",
+    "CHECKID",
```

Comment:
Is there any way to generate this headers dynamically using the `Check_Output` dataclass?

Reply:
You can use the following https://docs.python.org/3/library/dataclasses.html#dataclasses.asdict

Reply:
Already done